.github/workflows/deps.yml

```diff
@@ -30,14 +30,20 @@ jobs:
       # Run the commands to generate dependencies before and after and compare.
       - name: Compare dependencies
         run: |
-          BEFORE="$(mktemp -d)"
-          AFTER="$(mktemp -d)"
+          set -eux
```

Comment:
I would prefer not to have this (at least the `-x` (tracing) part) since the output _should_ generally be pretty human readable, and the commands should be pretty stable.

The reason I did `-x` in the vet script was mainly laziness around not wanting to output everything the script was doing before & after it did it.  (In hindsight that was probably a bad idea, as the output confuses most people.)